articles/fire-forests-inventories/index.md

```diff
@@ -0,0 +1,206 @@
+
+As a technical aside, we will use units of million (metric) tons of CO₂ to align with standard practices in greenhouse gas inventories. However, we note that the broader forest and climate community often reports results in units of Tg C (1 teragram of carbon), so we will convert results drawn from the literature into million metric tons of CO₂. A teragram of carbon is equal to about 3.67 million tons of CO₂, as CO₂ has a molecular weight about 3.67 times greater than carbon while million tons and teragrams are equivalent units.
+
+### What part(s) of the land will you study?
+
+Estimates of carbon fluxes depend on what part of the natural landscape you are discussing. For example, one could assess the carbon fluxes from all wildland ecosystems (e.g. all land outside of crops and urban areas) and derive whether those lands are a net source or sink. Alternatively, one could restrict the analysis to just trees without foliage (i.e. just the woody parts like trunks and branches). This decision will change the scope of the analysis.
+
+For the purposes of this article we will focus on the carbon balance in forests and compare estimates from the literature that best align with that definition. While a complete analysis would include lands not under tree cover, we justify our focus on forests since they hold 90% of the state’s aboveground carbon.<Cite id='gonzalez.2015'/>
+
+### What net emissions approach will you use?
+
+A forest ecologist can calculate a forest carbon flux of in a variety of different ways. In a 2019 paper, UC Berkeley researcher Tim Holland and colleagues offered a window into the options.<Cite id='holland.2019'/> The authors assessed two previous studies that independently calculated the net carbon flux from California’s forests and arrived at opposite results. The first, Gonzalez et al. (2015), used remotely-sensed land cover class analysis (Landfire) and found that trees in California’s forests were a source of 176 million tons of CO₂ in the 9 years between 2001 to 2010, or just under 20 million tons of CO₂ per year.<Cite id='gonzalez.2015'/> In contrast, the second study, Christensen et al. (2017), used in situ forest stock inventory estimates from the [Forest Inventory and Analysis (FIA) program](https://www.fia.fs.fed.us/) and concluded that California’s forests have been a net sink of 24 million tons of CO₂ per year.<Cite id='christensen.2017'/> These findings are based on standing live trees assessed by the AB 1504 California Forest Ecosystem and Harvested Wood Product Carbon Inventory, which is updated annually. The latest iteration for the [2019 reporting period](https://bof.fire.ca.gov/media/beddx5bp/6-final_forest_ecosys_hwp_c_2019_feb2021_all_ada.pdf) attributed to forests a net sink of 13 million tons of CO₂ per year.
```

Comment:
note that this `gonzalez.2015` citation might be a case where we want to use `hide` once https://github.com/carbonplan/layouts/pull/12 goes in

Reply:
yep! i thought of that too, but think it's fine for now